src/reconcile/rec_write.c

```diff
@@ -2943,6 +2944,7 @@ __rec_write_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
               F_ISSET(r, WT_REC_IN_MEMORY) ||
                 (F_ISSET(r, WT_REC_EVICT) && r->leave_dirty && r->multi->supd_entries != 0),
               "Attempting a 1-for-1 page swap when there are still updates to write");
+            __wt_atomic_addv16(&ref->ref_changes, 1);
```

Comment:
This is redundant as now we have increased before.

Reply:
@quchenhao Could you clarify where we had increased it? I put in multiple statements to increment the ref_changes as they are for different cases of the switch statement.

Reply:
We increase it in __rec_write_wrapup. They are in the same code path.

Reply:
Oh,  I misunderstood it. I thought the code on line 2902 is outside the switch call. We only need to increase once outside the switch call. We don't need it increase it inside the switch.